bin/public/js/term.js

```diff
@@ -1328,7 +1328,8 @@ Terminal.prototype.refresh = function(start, end) {
                 out += 'font-weight:bold;';
               }
               // See: XTerm*boldColors
-              if (fg < 8) fg += 8;
+              if(this.options.boldAsBright > 0 && fg < 8)
```

Comment:
True, I'm not a fan of webpack, but I like `browserify`. Will test your changes soon! :+1: